api/model/src/main/java/org/projectnessie/model/Validation.java

```diff
@@ -95,6 +95,24 @@ public final class Validation {
   public static final Pattern REF_NAME_PATH_ELEMENT_PATTERN =
       Pattern.compile(REF_NAME_PATH_ELEMENT_REGEX);
 
+  /*
+   * Default Cut-Off Policy can be a number or Duration or ISO instant.
+   * Following rules are to validate the same.
+   * */
+  private static final String DURATION_REGEX =
```

Comment:
question: what about year and month?

Reply:
I might have confused with seeing example of string representation of `Period`. 
Duration doesn't have years and month
https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toString--

Reply:
We are using same format for nessie-gc . Also it seems Java Duration doesn't support others. 
The same pattern I took from Java class. 
`
 private static class Lazy {
        static final Pattern PATTERN =
            Pattern.compile("([-+]?)P(?:([-+]?[0-9]+)D)?" +
                    "(T(?:([-+]?[0-9]+)H)?(?:([-+]?[0-9]+)M)?(?:([-+]?[0-9]+)(?:[.,]([0-9]{0,9}))?S)?)?",
                    Pattern.CASE_INSENSITIVE);
    }
`